@remirror/editor-social/src/components/social-editor.tsx

```diff
@@ -394,6 +425,26 @@ export class SocialEditor extends PureComponent<SocialEditorProps, State> {
             suggestionKeyBindings={this.emojiKeyBindings}
             onSuggestionExit={this.onEmojiSuggestionExit}
           />
+
+          {rich ? <RemirrorExtension Constructor={ParagraphExtension} /> : null}
+          {rich ? <RemirrorExtension Constructor={BoldExtension} /> : null}
+          {rich ? <RemirrorExtension Constructor={UnderlineExtension} /> : null}
+          {rich ? <RemirrorExtension Constructor={ItalicExtension} /> : null}
+          {rich ? <RemirrorExtension Constructor={BlockquoteExtension} /> : null}
+          {rich ? <RemirrorExtension Constructor={StrikeExtension} /> : null}
+          {rich ? <RemirrorExtension Constructor={CodeExtension} /> : null}
+          {rich ? <RemirrorExtension Constructor={HardBreakExtension} /> : null}
+          {rich ? (
+            <RemirrorExtension
+              Constructor={CodeBlockExtension}
+              supportedLanguages={supportedLanguages}
+              formatter={formatter}
+              syntaxTheme={syntaxTheme}
+              defaultLanguage={defaultLanguage}
+            />
+          ) : null}
+          {rich ? <RemirrorExtension Constructor={SSRHelperExtension} /> : null}
+
```

Comment:
I'm pretty sure this is achieveable by passing the `extensions` prop to `<SocialEditor` directly and having all this extension code instead live in `examples/with-next/pages/editor/rich-social/index.tsx`. That way we can demonstrate how to add extensions to a showcase editor without requiring changes to core.

However, I cannot figure out what the equivalent of these extension components is when passing to the `extensions` prop. Please can you advise?

Also, would it make sense to allow the `extensions` prop to accept these `<RemirrorExtension />` elements for consistency with the React API?